vnext/Shared/DevSupportManager.h

```diff
@@ -43,8 +47,15 @@ class DevSupportManager final : public facebook::react::IDevSupportManager {
       std::function<void()> onChangeCallback) override;
   virtual void StopPollingLiveReload() override;
 
+  virtual void startInspector(const std::string &packagerHost, const uint16_t packagerPort) override;
+  virtual void stopInspector() override;
+
  private:
   std::atomic_bool m_cancellation_token;
+
+#if defined(HERMES_ENABLE_DEBUGGER)
+  std::unique_ptr<InspectorPackagerConnection> m_InspectorPackagerConnection;
+#endif
```

Comment:
There's a single `DevSupportManager` around for the lifetime of the app, so I'm not sure it makes sense to keep this object to keep this state. `PackagerConnection` keeps a pool of connections alive forever, for different host names, etc.